chunky/src/java/se/llbit/chunky/chunk/BlockPalette.java

```diff
@@ -338,11 +338,14 @@ public static Map<String, Consumer<Block>> getDefaultMaterialProperties() {
     materialProperties.put("minecraft:redstone_torch", block -> {
       block.emittance = 1.0f;
     });
+    materialProperties.put("minecraft:redstone_wall_torch", block -> {
+      block.emittance = 1.0f;
+    });
```

Comment:
Now that I see it… shouldn't the torches take their on/off state into account? Is this a bug you'd like to fix?

Reply:
Maybe I can try fixing that in a future PR.